StyleGuide.md

```diff
@@ -206,124 +322,142 @@ Try to address as many of the suggestions as possible. If Vale flags a word that
 
-Each version's pages are found in a directory named for the version. For example, pages for CockroachDB v21.1 are in the `docs > v21.1` directory. For more information about page structure, see the [Pages](https://github.com/cockroachdb/docs/blob/main/CONTRIBUTING.md#pages) section in our [Contributing Guide](https://github.com/cockroachdb/docs/blob/main/CONTRIBUTING.md). For more information about how to style page content, see [Components](#components).
+## Page components
+
+This section describes how to approach page elements other than body text. For Markdown-specific syntax and formatting guidelines, refer to the [Markdown Guide](MarkdownGuide.md).
 
-Each version's images are stored in a versioned directory under the `images` directory. For example, images for CockroachDB v21.1 are in the `docs > images > v21.1` directory. For more information, see [Images](#images).
+### Code
 
-### File naming
+#### Code blocks
 
-File names should match the page title. If you need to change a file name, it is necessary to do the following:
+Use separate code blocks for input and output. For example, rather than place both a SQL command and its resulting output within a single code block, place the SQL command inside one code block, followed by its output in another code block.
 
-- Add the previous page URL with the new URL to `_redirects`.
-- Replace all links to the previous file name with the new file name in the applicable docs versions.
+When you use multiple code blocks to demonstrate a task, interleave the code blocks with text that explains the content of each code block and clarifies where it stands in the overall flow.
 
-## Product names
+**Example:**
 
-All product names except CockroachDB should be written as Liquid variables unless part of front-matter, file names, or non-Markdown files. Use the following code in place of product names:
+```
+1. To view all virtual clusters on the standby, run:
 
-- **CockroachDB Cloud** : `CockroachDB {{ site.data.products.cloud }}`
-- **CockroachDB Basic** : `CockroachDB {{ site.data.products.basic }}`
-- **CockroachDB Standard** : `CockroachDB {{ site.data.products.standard }}`
-- **CockroachDB Advanced** : `CockroachDB {{ site.data.products.advanced }}`
-- **self-hosted** : `CockroachDB {{ site.data.products.core }} cluster`
+    {% include_cached copy-clipboard.html %}
+    ~~~ sql
+    SHOW VIRTUAL CLUSTERS;
+    ~~~
 
-The first occurrence of a product name within a docs page should use its full name. At the writer's discretion, subsequent occurrences may be shortened to "Basic", "Advanced", or "Cloud", unless a writer (or reviewer) senses contextual ambiguity that could be improved by using the full product name. In long pages, it may be helpful to use the full name for each occurrence in a new sentence or if it's been a few paragraphs since an occurrence of the full product name.
+    The standby cluster will show the `main` virtual cluster is in a `replicating` state.
 
-## Page components
```

Comment:
i think this was a separate PR but i am kinda bummed that we removed the stuff about consistent page structure, e.g. 

- before you begin
- ...
- see also

since i think there is a lot of value of having ~all the pages in the docs share a rough structure for easier scanning, since folks aren't usually "reading" carefully but "scanning"

however maybe this doesn't matter in agent world (tm)

Reply:
I don't recall seeing or removing those sections. I see that some TOC links corresponding to those were removed in https://github.com/cockroachdb/docs/pull/20170, but I think that content was previously removed (not sure if by me). I'll dig around for it in the git history.

Reply:
looks like it was removed in this PR: https://github.com/cockroachdb/docs/pull/19334/files